librz/core/cdebug.c

```diff
@@ -590,3 +590,58 @@ RZ_API void rz_debug_map_list_visual(RzDebug *dbg, ut64 addr, const char *input,
 		print_debug_maps_ascii_art(dbg, maps, addr, colors);
 	}
 }
+
+/**
+ * Print all traces
+ * @param dbg core->dbg
+ * @param mode output mode, default RZ_OUTPUT_MODE_STANDARD
+ * @param offset offset of address
+ */
+RZ_API void rz_debug_trace_print(RzDebug *dbg, RzOutputMode mode, ut64 offset) {
+	rz_return_if_fail(dbg);
+	int tag = dbg->trace->tag;
+	RzListIter *iter;
+	RzDebugTracepoint *trace;
+	switch (mode) {
+	case RZ_OUTPUT_MODE_QUIET:
+		rz_list_foreach (dbg->trace->traces, iter, trace) {
+			if (!trace->tag || (tag & trace->tag)) {
```

Comment:
move these two lines outside of the switch, so you don't need to repeat them for each case.